lib/Dependency.js

```diff
@@ -182,6 +182,13 @@ class Dependency {
 		this._loc = undefined;
 	}
 
+	/**
+	 * @returns {string | undefined} a request context
+	 */
+	getContext() {
```

Comment:
we can't use `get context() {}` `context` property is used in mini-css ( so this will error on set )